Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,21 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  url "https://github.com/MariaDB/mariadb-connector-odbc/archive/3.0.1.tar.gz"
```

Comment:
Is there a release tarball at `downloads.mariadb.org`?

Reply:
It's behind a login/account creation. 

The same goes for the mariadb-connector-c, but the original formula isn't mine and already used a direct link. I'm not sure if mariadb is actually okay with that, but I just kept it that way.

Reply:
If we can get the direct download link, we should just use that.

Reply:
ok, took one of the mirror links, downloads.mariadb.org itself just redirects to mirrors

Reply:
Turns out taking a direct URL gives me this:

Stable version URLs should not contain beta

http://mariadb.mirror.triple-it.nl/connector-odbc-3.0.1/mariadb-connector-odbc-3.0.1-beta-src.tar.gz

Problem is, 3.0.2 doesn't actually build (It relies on an unreleased version of a dependency) and there's nothing else to build to get a OSX odbc connector